app/server/services/stats.js

```diff
@@ -4,7 +4,7 @@ var User = require('../models/User');
 
 // In memory stats.
 var stats = {};
-function calculateStats(){
+function calculateStats(filterField){
```

Comment:
For this stats function, if I am not mistaken, it is used by the main admin page to show stats. Make sure that this does not affect the behavior of that statistics page. If you change this, you need to change everywhere this function is called. The places that should also be changed are the end of stats.js file and potentially the user controller